main/templates/main/messages.html

```diff
@@ -0,0 +1,30 @@
+{% load tz %}
```

Comment:
Don't think this is needed.

Reply:
There's a `{% now "e (O)" as server_time %}` that I think needs this.

Reply:
I don't think `now` is one of the tags provided by `tz` - https://docs.djangoproject.com/en/5.1/topics/i18n/timezones/#template-tags

Reply:
Nothing *seems* to break if I remove it. I'll take it out for now.